plugin.xml

```diff
@@ -99,21 +99,21 @@
     <framework src="CoreTelephony.framework"/>
     <framework src="EventKit.framework"/>
     <framework src="EventKitUI.framework"/>
+    <framework src="MediaPlayer.framework"/>
     <framework src="MobileCoreServices.framework"/>
     <framework src="QuartzCore.framework"/>
     <framework src="Security.framework"/>
     <framework src="Social.framework"/>
     <framework src="StoreKit.framework"/>
+    <framework src="SystemConfiguration.framework"/>
     <framework src="iAd.framework"/>
 
     <framework src="libxml2.dylib"/>
+    <framework src="libsqlite3.dylib" /> 
```

Comment:
are you using Xcode 7? Apple decided to change the extension of these libs to `.tbd` which is annoying. Just curious if you've tried this with Xcode 7, it may be worth a line in the README since there might not be a nicer solution than having people change it